README.md

```diff
@@ -1,3 +1,7 @@
+> :warning: In the future, [the `senna` npm package](https://npmjs.com/package/senna) will be deprecated and this repo archived (ie. switched to read-only mode). Development continues in [the `liferay/liferay-portal` repo](https://github.com/liferay/liferay-portal) (more specifically, inside [the `frontend-js-spa-web` module](https://github.com/liferay/liferay-portal/tree/master/modules/apps/frontend-js/frontend-js-spa-web)), but not as an independent project, in order to better integrate with and serve the needs of Liferay DXP.
+
+---
```

Comment:
Let's add also something like...

> The `2.x` version is the last official release blabla... if you're interested in becoming a maintainer to improve this let us know or for it or I don't know, something!!

#justsaying

Reply:
![anyone](https://user-images.githubusercontent.com/7074/100764764-fe848080-33f6-11eb-9c70-1cce7abab92d.gif)

One thing to bear in mind is that from a legal perspective, we open ourselves up to liability by allowing a third-party to take ownership (or even just write privileges) on a package that is (or was) associated with the Liferay brand. People may have `senna` in their package.json files with a lose enough version range that they'll unwittingly get a copy of the library from an entity other than the one they initially chose to trust...

`#justsaying`

Reply:
Fair enough, let's then simply state that 

> `2.x` is the last official release version that will be used only for maintenance as long as DXP versions that use it haven't reached EOL. 

If anyone is really interested in picking this up, they can let us know and we can figure out what that really means.

How does that sound?